src/test/statistics/chunk_statistics/counting_quotient_filter_test.cpp

```diff
@@ -91,14 +91,43 @@ class CountingQuotientFilterTest : public BaseTest {
   }
 
   template <typename DataType>
-  void test_can_prune(std::shared_ptr<CountingQuotientFilter<DataType>> cqf, std::map<DataType, size_t>
+  void test_can_not_prune(std::shared_ptr<CountingQuotientFilter<DataType>> cqf, std::map<DataType, size_t>
       value_counts) {
     for (auto value_and_count : value_counts) {
         EXPECT_FALSE(cqf->can_prune(value_and_count.first, PredicateCondition::Equals));
     }
   }
+
+  template <typename DataType>
+  DataType get_test_value(int run) {
+    throw std::logic_error("specialize this method");
+  }
+
+  template <typename DataType>
+  void test_false_positive_rate(std::shared_ptr<CountingQuotientFilter<DataType>> cqf) {
+    int runs = 1000;
+    int false_positives = 0;
+    for (int run = 0; run < runs; run++) {
```

Comment:
`size_t` for the loop variable or Moritz is going to hunt you down!

Reply:
Done.